schedule/config.go

```diff
@@ -69,7 +68,7 @@ func (s *ScheduleConfig) GetPriority() string {
 	return s.Priority
 }
 
-func (s *ScheduleConfig) GetLockMode() ScheduleLockMode {
+func (s *Config) GetLockMode() ScheduleLockMode {
 	switch s.LockMode {
 	case constants.ScheduleLockModeOptionFail:
 		return ScheduleLockModeFail
```

Comment:
I think we forgot to wire up the schedule lock options to generate command line flags 🤔 I can't see anything using the `GetLockMode()` method...